code/modules/projectiles/ammo_boxes/misc_boxes.dm

```diff
@@ -138,3 +138,31 @@
 
 /obj/item/ammo_box/magazine/misc/flares/empty
 	empty = TRUE
+
+//------------------------Flashlight Box--------------------------
+
+/obj/item/ammo_box/magazine/misc/flashlight
+	name = "box of flashlights"
+	desc = "A box of flashlights to brighten your day!"
+	magazine_type = /obj/item/device/flashlight
+	num_of_magazines = 8
```

Comment:
should probably renamed this var to num_of_objects

Reply:
That can be done in a separate refactor since the parent objects do use `num_of_magazines` as a way of getting the correct sprites and all that.